File: src/main/java/nl/guno/intellij/savechangelisttopatches/SaveChangeListsToPatchesAction.java
package nl.guno.intellij.savechangelisttopatches;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.actionSystem.CommonDataKeys;
import com.intellij.openapi.project.Project;

public class SaveChangeListsToPatchesAction extends AnAction {
    @Override
    public void actionPerformed(AnActionEvent e) {
        Project project = e.getData(CommonDataKeys.PROJECT);

        SaveChangeListsToPatchesTask task = new SaveChangeListsToPatchesTask(project);
        task.queue();
    }
}

File: src/main/kotlin/nl/guno/intellij/savechangelisttopatches/SaveChangeListsToPatchesAction.kt
package nl.guno.intellij.savechangelisttopatches

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.actionSystem.CommonDataKeys

class SaveChangeListsToPatchesAction : AnAction() {
    override fun actionPerformed(e: AnActionEvent) {
        val project = e.getData(CommonDataKeys.PROJECT)

        val task = SaveChangeListsToPatchesTask(project)
        task.queue()
    }
}
